client/pages/editTemplate.jsx

```diff
@@ -0,0 +1,247 @@
+import React, { useEffect, useState } from "react";
+import {
+  addTemplate,
+  saveTemplateName,
+  saveTemplatePhases,
+  deleteTemplate,
+  getTemplateByID,
+  getTemplates
+} from "../utils/apiWrapper";
+import { Head, TemplateDraft } from "../components";
+import {
+  Button,
+  Row,
+  Container,
+  Alert,
+  Form,
+  FormGroup,
+  Input,
+  Label
+} from "reactstrap";
+import "../public/styles/editTemplate.scss";
+
+export default function EditTemplate() {
+  const [name, setName] = useState("");
+  const [mounted, setMounted] = useState(false);
+  const [templateID, setTemplateID] = useState("");
+  const [error, setError] = useState(false);
+  const [templates, setTemplates] = useState([]);
+  const [isInspiration, setInspiration] = useState(false);
+  const [isIdeation, setIdeation] = useState(false);
+  const [isImplementation, setImplementation] = useState(false);
```

Comment:
would it make more sense to combine these into a single state? maybe an object with keys for the phases or an array containing the selected phases?

Reply:
Left as separate states.